firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/Logger.java

```diff
@@ -33,65 +35,65 @@ public static Logger getLogger() {
     return DEFAULT_LOGGER;
   }
 
-  public boolean isLoggable(String tag, int level) {
+  private boolean canLog(int level) {
```

Comment:
This name had to change because there is a heuristic lint check, `LogTagMismatch` that was getting confused by the lack of tag